src/charts/bar.js

```diff
@@ -1292,6 +1302,15 @@ define(function(require) {
             return this;
         };
 
```

Comment:
We would need a comment on this accessor so we generate the documentation

Reply:
Added it